README.md

```diff
@@ -14,7 +14,8 @@ endpoint: AAD v1.0
 
 ## About this sample
 
-This sample shows how to build an ASP.NET MVC web application that uses Azure AD for sign-in using the OpenID Connect protocol. Additionally it also introduces developers to the concept of a [multi-tenant](https://docs.microsoft.com/en-us/azure/active-directory/develop/single-and-multi-tenant-apps) Azure Active Directory application.
+This sample shows how to build an ASP.NET MVC web application that uses Azure AD for sign-in using the OpenID Connect protocol in Azure Global and Microsoft National Cloud environments. Additionally it also introduces developers to the concept of a [multi-tenant](https://docs.microsoft.com/en-us/azure/active-directory/develop/single-and-multi-tenant-apps) Azure Active Directory application.
```

Comment:
and Microsoft National Cloud environments => with a variation to explain how to do it in Microsoft National Cloud environments?

Reply:
Or even:

This sample shows how to build an ASP.NET MVC web application that uses Azure AD for sign-in using the OpenID Connect protocol. Additionally it also introduces developers to the concepts of:
- a [multi-tenant](https://docs.microsoft.com/en-us/azure/active-directory/develop/single-and-multi-tenant-apps) Azure Active Directory application.
- Azure Global and Microsoft National Cloud environments

Reply:
Deploying to a gov cloud is an orthogonal topic, that we can repeat for nearly all samples. We have a dedicated pages that introduce national clouds, the readme in the bottom references them. I'll  leave this text as is.